src/services/redisSessionStorage.ts

```diff
@@ -308,6 +308,19 @@ export default class RedisSessionStorage extends RedisStorageUtils
         )
       );
     }
+
+    // Remove SESSIONINFO reference from USERSESSIONS Set
+    // this operation is executed in background and doesn't compromise
+    // the logout process.
+    this.redisClient.srem(
```

Comment:
can this call throw? I'd add a `catch()` clause with a log since any uncaught exception in a promise will stop the crash the whole process